mmv1/templates/terraform/examples/storage_folder_basic.tf.tmpl

```diff
@@ -0,0 +1,21 @@
+resource "google_storage_bucket" "bucket" {
+  name                        = "{{index $.Vars "bucket_name"}}"
+  location                    = "EU"
+  uniform_bucket_level_access = true
+  hierarchical_namespace {
+    enabled = true
+  }
+  force_destroy = true
```

Comment:
To be more concise and clear about the functionality, would it make sense to remove `force_destroy = true` from the bucket and subfolder? I don't believe they are needed.

Reply:
removed from bucket and subfolder